### PR TITLE
Various improvements in config docs

### PR DESCRIPTION
**Pull request by obo**
Please check especially, if comments can appear anywhere, or if they need to be on separate lines.

### CODE REVIEW

**Comment by jindrahelcl** [comment]
Tady bude náhled, až se zbuildí
http://neural-monkey.readthedocs.io/en/config-docs-foolproofing

File: docs/source/configuration.rst [hunk 1]
```diff
@@ -42,27 +47,33 @@ The atomic values are:
...
 
-On top of that there are two compound types syntax from Python:
+On top of that, there are two compound types syntax from Python:
 
 -  lists: comma-separated in squared brackets (e.g., ``[1, 2, 3]``)
 
 -  tuples: comma-separated in round brackets (e.g.,
    ``("target", <ter>)``)
 
-We also require to have all key-value pairs in sections.
```
Comment (jindrahelcl) [inline docs/source/configuration.rst]: proč jsi dal tohle pryč?

Comment (obo) [inline docs/source/configuration.rst]: Napsal jsem stejnou informaci v textu vyse, podle mne lepe zduraznenou.

Comment (jindrahelcl) [inline docs/source/configuration.rst]: jo jasný, už to vidim; prve jsem to přehlédnul.

File: docs/source/configuration.rst [hunk 2]
```diff
@@ -9,21 +9,26 @@ is run.
...
+Every section starts with its header which consists of the section
 name in square brackets. Everything below the header is considered a
-part of the section. The files can also contain comments starting with
-semi-colons (``;``). You can find more details on INI syntax, e.g., on
-the corresponding `Wikipedia
-page <https://en.wikipedia.org/wiki/INI_file>`__.
+part of the section.
+
+Comments start with semi-colons (``;``) and they can appear at the
+beginning or anywhere else in the line (except string literals, of course).
```
Comment (jindrahelcl) [inline docs/source/configuration.rst]: tohle neni pravda
https://docs.python.org/3/library/configparser.html

Comment (jindrahelcl) [inline docs/source/configuration.rst]: opravuju

**Comment by obo** [comment]
Preformuloval bys to prosim? Ukrutne nestiham a nechci, aby to zbytecne dlouho viselo.

----- Original Message -----
> From: "Jindra Helcl" <notifications@github.com>
> To: "ufal/neuralmonkey" <neuralmonkey@noreply.github.com>
> Cc: "Ondrej Bojar" <bojar@ufal.mff.cuni.cz>, "Author" <author@noreply.github.com>
> Sent: Friday, 20 January, 2017 12:18:24
> Subject: Re: [ufal/neuralmonkey] Various improvements in config docs (#265)

> jindrahelcl requested changes on this pull request.
> 
> 
> 
>>  name in square brackets. Everything below the header is considered a
> -part of the section. The files can also contain comments starting with
> -semi-colons (``;``). You can find more details on INI syntax, e.g., on
> -the corresponding `Wikipedia
> -page <https://en.wikipedia.org/wiki/INI_file>`__.
> +part of the section.
> +
> +Comments start with semi-colons (``;``) and they can appear at the
> +beginning or anywhere else in the line (except string literals, of course).
> 
> tohle neni pravda
> https://docs.python.org/3/library/configparser.html
> 
> --
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/ufal/neuralmonkey/pull/265#pullrequestreview-17656733

-- 
Ondrej Bojar (mailto:obo@cuni.cz / bojar@ufal.mff.cuni.cz)
http://www.cuni.cz/~obo

File: docs/source/configuration.rst (hunk 2 above)
Comment (jindrahelcl) [inline docs/source/configuration.rst]: hotovo

**Pull request merged**